chromium_src/components/crx_file/crx_creator.cc

```diff
@@ -0,0 +1,51 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * you can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "components/crx_file/crx_creator.h"
+
+namespace crx_file {
+class CrxFileHeader;
+
+std::string GetCrxId_BraveImpl(const std::string& key, CrxFileHeader* header);
```

Comment:
new line after?

Reply:
Done.
Not sure that CC has strict rules about this. I've tried to move the main include up to the begin.